### ISSUE TITLE
Upgrade to AF2 2.3.0

### ISSUE THREAD

**Issue by ajasja** [YoshitakaMo/localcolabfold#132]
**What is your question?**
There is a new version of AF2 (2.3.0) from Dec 13 2022 (https://github.com/deepmind/alphafold/releases/tag/v2.3.0),

Is there any specific extra work required to upgrade localcolabfold to 2.3?
Colab fold has a beta version here: https://github.com/sokrypton/ColabFold/issues/345

**Comment by sokrypton** [comment]
We are still doing some testing to make sure everything works as expected. Once testing is done (hopefully this week), we'll merge soon!

**Comment by mjw99** [comment]
This might be superseded by: https://github.com/deepmind/alphafold/releases/tag/v2.3.1

**Comment by YoshitakaMo** [comment]
Now ColabFold and local version have been updated! See also the [change log](https://github.com/sokrypton/ColabFold/wiki/v1.5.0) .
